cypress/integration/chi-js/chi-drawer.spec.js

```diff
@@ -0,0 +1,113 @@
+describe('chi-drawer', function() {
+  before(() => {
+    cy.visit('/tests/js/chi-drawer.html');
+  });
+
+  describe('check that click on "X" in left menu drawer, closes drawer', () => {
```

Comment:
Technically you are testing Open and Close as well. Let's tweak the context of the test group:
```suggestion
  describe('Test open & close behavior of Left positioned Drawer', () => {
```

And inside the describe statement let's add 2 test cases as split scenarios of Open and Close. Something like this:

```
describe('Test open & close behavior of Left positioned Drawer',()=>{
  it('Click on Drawer trigger opens the Drawer', () => {
    /*  */
  });

  it('Click on X button closes the Drawer', () => {
  /*  */
  });
});
```
Basically, split the following test assertions into 2: Open and Close.